backend/routes/campgrounds.js

```diff
@@ -3,8 +3,31 @@
 
     //GET all campgrounds
     router.route('/').get((req, res) =>{ 
-        Campground.find()
-            .then(campgrounds => res.json(campgrounds))
+
```

Comment:
I am a little worried about this code because in the latest commit I did the campground find is using paginated campgrounds to help with database load but im sure I can get this code working with what we got